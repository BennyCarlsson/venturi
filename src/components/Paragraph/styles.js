import styled from 'styled-components';

export const ParagraphWrapper = styled.div`
  padding: 16px;
`;
