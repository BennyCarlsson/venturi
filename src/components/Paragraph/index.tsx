import React from 'react';
import { ParagraphWrapper } from './styles';

type ParagraphProps = {
  title: string;
};

const Paragraph = ({ title, ...props }: ParagraphProps) => {
  return <ParagraphWrapper {...props}>{title}</ParagraphWrapper>;
};

export default Paragraph;
