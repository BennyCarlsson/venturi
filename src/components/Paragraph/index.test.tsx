import React from 'react';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Paragraph from '.';

describe('<Paragraph />', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<Paragraph title="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should renders without crashing', () => {
    render(<Paragraph title="test" />);
    expect(screen).toBeTruthy();
  });
});
