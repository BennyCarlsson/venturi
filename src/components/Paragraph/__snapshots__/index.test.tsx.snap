// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Paragraph /> should match snapshot 1`] = `
<div
  className="sc-bdfBwQ gDhytj"
>
  test
</div>
`;
